lib/Shuttle.js

```diff
@@ -25,16 +25,18 @@ class Shuttle extends EventEmitter {
   pids = pids
 
   // Methods
-  start () {
-    usb.on('attach', (d) => {
-      // Delay connection by 1 second because
-      // it takes a second to load on macOS
-      setTimeout(() => {
-        this._connect()
-      }, 1000)
-    })
-    // Find already
-    this._connect()
+  start (watchUsb = true) {
+    if (watchUsb) {
+      usb.on('attach', (d) => {
+        // Delay connection by 1 second because
+        // it takes a second to load on macOS and Linux
+        setTimeout(() => {
+          this._search()
```

Comment:
We could probably add a connect by usb device which short-circuits a bit of the searching when attach triggers.

Reply:
Maybe we could use the device returned by the attach event instead of doing the search.

Reply:
Actually, probably not, because no path info is given in that object.

Reply:
We could do various stuff like only check the one shuttleDef with a find as we know the VID/PID at least. We couldn't open the HID by VID/PID, but maybe we can match from hid.devices() via some other means, or move the connected paths check earlier in the logic, it's more expensive but might rule out more of them...

Reply:
It probably doesn't matter either way because this only happens when a device is connected, and I doubt someone other than those testing will be plugging in and unplugging devices in rapid succession.

Reply:
True, I'd rather not cause an edge case where if there are two on a hub only one gets opened if you're unlucky. The current code might be slightly less efficient for some cases but should at least be safe I think.